### PR TITLE
build: disable whitesource for submodules which are not to be published

### PR DESCRIPTION
**Pull request by jrudolph**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -251,11 +251,14 @@ lazy val httpTests = project("akka-http-tests")
 
 lazy val httpJmhBench = project("akka-http-bench-jmh")
   .settings(commonSettings)
+  .settings(
+    whitesourceIgnore := true
+  )
   .dependsOn(http, http2Support % "compile->compile,test")
   .addAkkaModuleDependency("akka-stream")
   .enablePlugins(JmhPlugin)
   .enablePlugins(NoPublish).disablePlugins(BintrayPlugin) // don't release benchs
```
Comment (jrudolph) [inline build.sbt:260]: We could also just add the setting to `NoPublish`, WDYT?

Comment (raboof) [reply build.sbt:260]: that makes sense to me :+1:

Comment (jrudolph) [reply build.sbt:257]: Done

**Comment by jrudolph** [comment]
Validating here: https://travis-ci.org/github/akka/akka-http/builds/737976439

**Comment by jrudolph** [comment]
Finally fixed :)

**Pull request merged**